### DISCUSSION TITLE
I'm having trouble giving Monika the promise ring in MAS

### DISCUSSION THREAD

**Discussion by WLbEoDQ** [Monika-After-Story/MonikaModDev#8282] [QA - tech support + general] [answered]
I have gifted the ring 2 times earlier  but at that time i did not have 400+ affection but now if i give it to her she wont react to the it but is able to detect other gifts , there is no typo error , i have 400+ affection also, i placed the gift file in correct folder, she also bought up the topic of promise ring.

**Comment by Booplicate** [comment, marked as answer]
A few possible things could happen:
- you're using a translation, they're known to have a few issues
- you've timetravelled and tried to gift it in the future, thereby locked yourself from gifting the ring until that date
- you tried to gift it today and she didn't accept it, then you can wait until tomorrow to try again
